### PR TITLE
Release version 1.2

### PR DESCRIPTION
**Pull request by rachelbaker**
Bumps the version number, updates the project name, and author info, and includes the 1.2 changelog.

### CODE REVIEW

**Comment by rachelbaker** [comment]
@rmccue #reviewmerge specifically the ordering and descriptions in the changelog.  

Then we need to tag the release, and push to svn.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,325 @@
...
+
+- Update the cookie authentication documentation examples.
+
+  With 1.1 the localized JavaScript object for `wp-api.js` changed to
+  `WP_API_Settings`. This updates the Authentication section documentation
+  nonce example to use the updated object name.
+
+  (props @rachelbaker, [#321][gh-321])
+
+- Add flexibly and multisite support to unit tests.
```
Comment (rmccue) [inline CHANGELOG.md]: Should be "flexibility"

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,325 @@
...
+
+  (props @ircrash, @rmccue, [#405][gh-405])
+
+- Register the JavaScript client in the admin.
+
+  (props @tlovett1, [#473][gh-473])
+
+- Remove duplicate error checks on post ids.
+
+	(props @danielbachhuber, [#271][gh-271])
```
Comment (rmccue) [inline CHANGELOG.md]: Indentation issue

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,325 @@
...
+[gh-347]: https://github.com/WP-API/WP-API/issues/347
+[gh-378]: https://github.com/WP-API/WP-API/issues/378
+[gh-528]: https://github.com/WP-API/WP-API/issues/528
+[gh-595]: https://github.com/WP-API/WP-API/issues/595
+[gh-730]: https://github.com/WP-API/WP-API/issues/730
+[gh-481]: https://github.com/WP-API/WP-API/issues/481
+[gh-524]: https://github.com/WP-API/WP-API/issues/524
+[gh-933]: https://github.com/WP-API/WP-API/issues/933
+[gh-474]: https://github.com/WP-API/WP-API/issues/474
+[gh-985]: https://github.com/WP-API/WP-API/issues/985
```
Comment (rmccue) [inline CHANGELOG.md]: Can we order these please? (Assuming you're in Sublime, select and Edit > Sort Lines should do)

**Comment by rmccue** [comment]
Looking good! Do you want me to write the release post, or you?

**Comment by rachelbaker** [comment]
@rmccue I have v1.2 tagged and ready for SVN push.  I can get the blog post out once the plugin version is updated.

**Pull request merged**